develop-docs/development-infrastructure/testing.mdx

```diff
@@ -13,24 +13,13 @@ to use the `--project` flag to separate local testing volumes from test suite
 volumes:
 
 ```shell
-# Turn off services for local testing.
-sentry devservices down
+# Bring up services for testing
+devservices up
```

Comment:
This seems like the opposite of what it was previously, is that intentional?

Reply:
Yes, we no longer support passing in test prefix, so people just need to bring services up as normal to run tests